examples/dev-kits/manager.js

```diff
@@ -0,0 +1,10 @@
+import { addons } from '@storybook/addons';
+import { themes } from '@storybook/theming';
+
+addons.setConfig({
+  theme: themes.dark,
+  mapper: ({ id, url }, { kind, ...rest }) => ({
```

Comment:
You can use this function to map incoming stories from external urls to your own storybook